src/bundle/Core/DependencyInjection/Security/PolicyProvider/PoliciesConfigBuilder.php

```diff
@@ -21,7 +21,17 @@ public function addConfig(array $config)
 
         // We receive limitations as values, but we want them as keys to be used by isset().
         foreach ($config as $module => $functionArray) {
+            if (preg_match('/\W+/', $module)) {
+                throw new \InvalidArgumentException(
+                    "Policy module '{$module}' can only contain characters [a-zA-Z0-9_]"
```

Comment:
I think this must be spelled out better for those who don't speak fluent regexp. Also some letters outside of a-z can be accepted depending on locale settings, like some accented letters, according to PHP's doc. It says "any letter or digit or the underscore character".

```suggestion
                    "Policy module '{$module}' can only contain letters, digits, or the underscore character."
```